### PR TITLE
chore: copy and paste available config

### PR DESCRIPTION
**Pull request by xiaoshihou514**

### CODE REVIEW

File: lua/guard-collection/init.lua [hunk 1]
```diff
@@ -0,0 +1 @@
+return true
```
Comment (glepnir) [inline lua/guard-collection/init.lua:1]: hmm what does this line do ?

**Comment by glepnir** [comment]
need ci for some test maybe.

File: lua/guard-collection/init.lua (hunk 1 above)
Comment (xiaoshihou514) [reply lua/guard-collection/init.lua:1]: So `pcall(require, "guard-collection")` tells one if it's available, or one would have to require guard-collection.formatter to do so

**Comment by xiaoshihou514** [comment]
> need ci for some test maybe.

You mean format_specs? True.

File: lua/guard-collection/init.lua (hunk 1 above)
Comment (glepnir) [reply lua/guard-collection/init.lua:1]: use `nvim_get_runtime_file`

**Comment by barrett-ruth** [comment]
Thanks for all this great work.

**Comment by xiaoshihou514** [comment]
@glepnir Could we land this and the corresponding pr for guard.nvim? Test cov is not 100% yet but I don't have many of the tools available so I can't really test them locally.

**Comment by glepnir** [comment]
There are some difficulties covering all the tools involved in the configuration of many environments. I think it's good enough for now.

**Comment by glepnir** [comment]
some empty files also need remove?

**Pull request merged**